plugins/available/autojump.plugin.bash

```diff
@@ -1,8 +1,13 @@
 cite about-plugin
 about-plugin 'Autojump configuration, see https://github.com/wting/autojump for more details'
 
-# Only supports the Homebrew variant at the moment.
+# Only covers MacOS brew and Debian installs
 # Feel free to provide a PR to support other install locations
 if command -v brew &>/dev/null && [[ -s $(brew --prefix)/etc/profile.d/autojump.sh ]]; then
-  . $(brew --prefix)/etc/profile.d/autojump.sh
+  source $(brew --prefix)/etc/profile.d/autojump.sh
+fi
+
+# Debian / Ubuntu - install it via apt-get install autojump
+if [[ -s /usr/share/doc/autojump/README.Debian ]]; then
```

Comment:
wanted to verify the existence of the autojump debian package (because that's the only one I tested for) instead of any arbitrarily installed autojump package, and the README is unique to the package. But yeah, could change it to the actual source file.